Blaise-Shyaka/Politico
# [#171720352] Create signup API endpoint

# What does this PR do?

It creates the signup API endpoint allowing users to create new accounts

# Description of tasks to be completed

- Setup Postgresql database
- Write a controller for the `/auth/signup`
- Mount it on a router
- Write tests for that endpoint

# How should this be manually tested?

- Clone this repository
- Run `npm install` to install all the dependencies
- Run `npm test` 
- 3 tests should be passing

# What are the relevant pivotal tracker stories?

#171720352
